iaso/tests/api/entities/test_entities.py

```diff
@@ -208,6 +208,148 @@ def test_list_entities_search_filter(self):
         self.assertEqual(len(response.json()["result"]), 1)
         self.assertEqual(the_result["id"], newly_added_entity.id)
 
+    def test_list_entities_search_uuid_filter(self):
+        """
+        Test the 'uuids:' search filter of /api/entities with comma-separated UUIDs
+        """
+        self.client.force_authenticate(self.yoda)
+
+        # Create multiple entities with different UUIDs
+        instance1 = Instance.objects.create(
+            org_unit=self.ou_country,
+            form=self.form_1,
+            json={"name": "Entity 1", "age": 25},
+        )
+        entity1 = Entity.objects.create(
+            name="Entity 1",
+            entity_type=self.entity_type,
+            attributes=instance1,
+            account=self.yoda.iaso_profile.account,
+        )
+
+        instance2 = Instance.objects.create(
+            org_unit=self.ou_country,
+            form=self.form_1,
+            json={"name": "Entity 2", "age": 30},
+        )
+        entity2 = Entity.objects.create(
+            name="Entity 2",
+            entity_type=self.entity_type,
+            attributes=instance2,
+            account=self.yoda.iaso_profile.account,
+        )
+
+        instance3 = Instance.objects.create(
+            org_unit=self.ou_country,
+            form=self.form_1,
+            json={"name": "Entity 3", "age": 35},
+        )
+        entity3 = Entity.objects.create(
+            name="Entity 3",
+            entity_type=self.entity_type,
+            attributes=instance3,
+            account=self.yoda.iaso_profile.account,
+        )
+
+        # Test single UUID search
+        response = self.client.get(f"/api/entities/?search=uuids:{entity1.uuid}", format="json")
```

Comment:
Don't forget to always test that a response has the expected HTTP status code (200 OK here).

This looks a bit superfluous but this simple test can catch so many problems for us.

You should do that for every response.

```suggestion
        response = self.client.get(f"/api/entities/?search=uuids:{entity1.uuid}", format="json")
        self.assertEqual(response.status_code, 200)
```